scripts/data/alloprof/create_data_reranking.py

```diff
@@ -0,0 +1,76 @@
+import datasets
+from sentence_transformers import SentenceTransformer, util
+import torch
+from huggingface_hub import create_repo
+from huggingface_hub.utils._errors import HfHubHTTPError
+
+"""
+To create a reranking dataset from the initial retrieval dataset, we use a model to embed
+the queries and the documents. We then compute the cosine similarity for each query and document.
+For each query we get the topk articles, as we would for a retrieval task.
+Each couple query-document is labeled as relevant if it was labeled like so in the retrieval dataset,
+or irrelevant if it was not
+"""
```

Comment:
Can you just mention the name of the model used to embed documents in the description (for both Alloprof and syntec) ? I guess it's this one : 'sentence-transformers/all-MiniLM-L6-v2'.

Reply:
Tout a fait ! Bien vu, je rajoute ça :)